kubevirt-cdi/step1.md

```diff
@@ -0,0 +1,51 @@
+### Wait for the Kubernetes cluster to be ready
+
+Before we can start, we need to wait for the Kubernetes cluster to be ready (a command prompt will appear once it's ready).
+
+#### Deploy KubeVirt
+
+In this scenario we'll go quickly over installing KubeVirt as it was covered already in the introduction scenario: 'First steps with KubeVirt'
+
+We query GitHub's API to get the latest available release (click on the text to autoexecute the commands on the console):
+
+`export KUBEVIRT_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt/releases/latest | jq -r .tag_name)
+echo $KUBEVIRT_VERSION`{{execute}}
+
+Run the following command to deploy the KubeVirt Operator:
+
+`kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-operator.yaml`{{execute}}
+
+This demo environment already runs within a virtualized environment, and in order to be able to run VMs here we need to pre-configure KubeVirt so it uses software-emulated virtualization instead of trying to use real hardware virtualization.
+
+`kubectl create configmap kubevirt-config -n kubevirt --from-literal debug.useEmulation=true`{{execute}}
+
+Now let's deploy KubeVirt by creating a Custom Resource that will trigger the 'operator' reaction and perform the deployment:
+
+`kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml`{{execute}}
+
+Let's check the deployment:
+`kubectl get pods -n kubevirt`{{execute}}
+
+Once it's ready, it will show something similar to:
```

Comment:
Advise the user that will take some time and he can repeat the command above to check the status.